File: Array/easy/1672. Richest Customer Wealth/maximumWealth.cpp
//time complexity : O(n+m)
//space complexity : O(1)
class Solution 
{
public:
    //in this solution i loop in the 2dvector and sum all elements in that index and find the max index
    int maximumWealth(vector<vector<int>>& accounts) 
    {
        int max = 0;
        for (vector<int> i : accounts)
        {
            int moneywealth = 0;
            for (int j : i)
            {
                moneywealth += j;
            }
            if (moneywealth > max)
            {
                max = moneywealth;
            }
        }
        return max;
    }
};

File: Array/easy/1672. Richest Customer Wealth/maximumWealth.py
#time complexity : O(n)
#spce complexity : O(1)
class Solution(object):
    #in this solution i loop in the list and sum all elements in that index and find the max index
    def maximumWealth(self, accounts):
        max = 0
        for i in accounts:
            money = sum(i)
            if money > max:
                max = money
                
        return max